src/test/java/core/basesyntax/service/FileWriterImplTest.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private final FileWriter fileWriter = new FileWriterImpl();
+
+    @Test
+    public void writeReport_existingPath_ok() {
+        fileWriter.writeReport("", "src/test/resources/TestWriter.csv");
```

Comment:
```suggestion
        fileWriter.writeReport("", OUTPUT_PATH);
```
AND, can we test the outcome of this method? compare the actual content?